### DISCUSSION TITLE
SOLVED: bank number confusion.

### DISCUSSION THREAD

**Discussion by toreriksolem** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#68] [Editor App Feature Requests] [answered]
This now works as expected!

This is a double error, really:

First error: When setting up a Go To Bank action, selecting for instance bank 19 from the drop down list, it will be set to bank 20.

Second error: Then when executing/pressing the switch, it jumps to bank 19 after all.

Confusing.

MacBook
 OS 11.5.1 
Running web app in Chrome Version 104.0.5112.79
USB-C cable directly to Bridge 6

**Comment by simonaglover95** [comment]
Is this what you're referring to? I just found this:
<img width="627" alt="Screen Shot 2022-08-16 at 3 23 44 pm" src="https://user-images.githubusercontent.com/59382082/184857630-dd81861d-757a-4260-a2a1-b4c4c61422fb.png">

**Reply by simonaglover95** [reply]
I tried to check again, and it isn't doing this now :/

**Comment by toreriksolem** [comment]
I was referring to this behaviour:

Image on the left, I have selected bank 20. Then when pressing SAVE to confirm, the window closes, and in the other window (my picture on the right) it says bank 21.

To make further confusion, when pressing this switch IRL, it actually takes me to bank 20, not 21.
![bank confusion](https://user-images.githubusercontent.com/69301387/184882881-50030f15-5992-4308-8b2b-6d777aafa7c7.jpg)

**Comment by toreriksolem** [comment]
Trying to recreate this behaviour now, but it seems that it has been fixed the last hours. Great work!

**Comment by konnsim** [comment, marked as answer]
The fix for this was bundled in with the hotfix the other day. Can close.
